Makefile

```diff
@@ -40,7 +40,7 @@ test-docstrings:
 	testimony validate_docstring tests/foreman/ui
 
 test-robottelo:
-	$(NOSETESTS) $(ROBOTTELO_TESTS_PATH)
+	$$(which nosetests) $(ROBOTTELO_TESTS_PATH)
```

Comment:
For the sake of completeness, this is what make will execute:

    $(which nosetests) tests/robottelo/

There's significant differences between make and shell syntax. Make considers the following to be a variable, and (ba)sh considers the following to be some command which should be executed in a sub-shell:

    $(something)